### DISCUSSION TITLE
builtins.NoneType' is not mapped

### DISCUSSION THREAD

**Discussion by Clives-online** [sqlalchemy/sqlalchemy#7560] [Usage Questions]
Greetings,

I am getting the following error, any suggestions would be welcomed.
When running: 

deleteVersionStamp     self.session.delete(rec)   File “\Externals\win32\Lib\site-packages\sqlalchemy\orm\scoping.py”, line 162,

in do     return getattr(self.registry(), name)(*args, **kwargs)   File " \Externals\win32\Lib\site-packages\sqlalchemy\orm\session.py", line 2014,

in delete     util.raise_(   File " \Externals\win32\Lib\site-packages\sqlalchemy\util\compat.py", line 178, in raise_     raise exception sqlalchemy.orm.exc.UnmappedInstanceError: Class 'builtins.NoneType' is not mapped

Class 'builtins.NoneType' is not mapped
Is this a permissions error or something else??

Thanks,
Clive

**Comment by CaselIT** [comment]
Hi,

it seems that `rec` is None. The error is trying to communicate that
